policyengine_us/variables/gov/local/co/denver/dhs/property_tax_relief/eligibility/co_denver_property_tax_relief_homeowner_eligible.py

```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class co_denver_property_tax_relief_homeowner_eligible(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Eligible for the homeowner Denver Property Tax Relief"
+    defined_for = "in_denver"
+    definition_period = YEAR
+    reference = "https://library.municode.com/co/denver/codes/code_of_ordinances?nodeId=TITIIREMUCO_CH53TAMIRE_ARTXIREPRTAASELLCOPROWTE_S53-492DE"  # 53-495 (d)
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(period).gov.local.co.denver.dhs
+        pays_property_taxes = add(spm_unit, period, ["real_estate_taxes"]) > 0
+        has_elderly_or_disabled = spm_unit(
+            "has_co_denver_dhs_elderly_disabled", period
+        )
+        count_dependents = add(spm_unit, period, ["tax_unit_dependents"])
+        has_dependents = count_dependents > 0
+        homeowners_with_elderly_or_disabled_or_dependents = (
+            has_elderly_or_disabled | has_dependents
+        ) & pays_property_taxes
+
+        income = spm_unit("co_denver_property_tax_relief_income", period)
+        size = spm_unit("spm_unit_size", period)
+        ami = spm_unit.household("ami", period)
+        p_hud = parameters(period).gov.hud.ami_limit
```

Comment:
could we refactor the file this logic is from to avoid duplication? maybe create separate variables for each of the limits, and use the moderate one here? especially since also in the renter variable

Reply:
Can I combine those variables into one, like my current version? The downside of my current code is I have to input hud_income_level = MODERATE in test cases.